pipelines/batch/Dockerfile

```diff
@@ -12,13 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM openjdk:8
+# If you are running locally, make sure to specify the --network=cloudbuild flag
+# and mount the openmrs-fhir-analytics directory to /workspace 
+# Example (run from the openmrs-fhir-analytics directory, not pipelines/batch):
+#  docker run -it -v "$(pwd):/workspace" --network=cloudbuild  \
+#  -e JDBC_MODE_ENABLED=true \
+#  -e FHIR_DEBEZIUM_CONFIG_PATH=/workspace/utils/dbz_event_to_fhir_config.json \   
+#  gobbledegook/batch-pipleine
```

Comment:
Can we wrap all of these into a script used for local runs? Also I think the e2e related setup warrants a short doc; please consider updating [`doc/developers.md`](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/blob/master/doc/developers.md), in particular the e2e section.

Reply:
Added a new script and modified the developers doc